tests/apm/monitoring-topologyOpening-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+MongoDB\Driver\Monitoring\TopologyOpeningEvent
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$m = create_test_manager();
+
+class MySubscriber implements MongoDB\Driver\Monitoring\SDAMSubscriber
+{
+    public function topologyChanged(MongoDB\Driver\Monitoring\TopologyChangedEvent $event) {}
+
+    public function topologyOpening(MongoDB\Driver\Monitoring\TopologyOpeningEvent $event)
+    {
+        echo "- getTopologyId() returns an ObjectId: ", ($event->getTopologyId() instanceof MongoDB\BSON\ObjectId) ? 'yes' : 'no', "\n";
+    }
+}
+
+$subscriber = new MySubscriber;
+MongoDB\Driver\Monitoring\addSubscriber($subscriber);
+
+$command = new MongoDB\Driver\Command(['ping' => 1]);
+$m->executeCommand(DATABASE_NAME, $command);
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECT--
+- getTopologyId() returns an ObjectId: yes
```

Comment:
I realize this wasn't done in the previous PR for TopologyChangedEvent, but it occurred to me that we have no tests (and therefore no code coverage) for the event object's debug handler. Would you mind making a ticket within the epic so we don't forget to add those? Something like "Create tests for event object debug handlers" should suffice.

Reply:
https://jira.mongodb.org/browse/PHPC-1953